dev-python/mongomock/mongomock-3.19.0.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+#DISTUTILS_USE_SETUPTOOLS=bdepend
+PYTHON_COMPAT=( python3_{6,8} )
+DISTUTILS_IN_SOURCE_BUILD=1
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="Library to help testing Python code that interacts with MongoDB via Pymongo."
+HOMEPAGE=" https://github.com/mongomock/mongomock https://pypi.org/project/mongomock/"
+SRC_URI="https://github.com/mongomock/mongomock/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+RESTRICT="test" # it has a lot of network tests
+
+RDEPEND="
+	>=dev-python/pymongo-3.10.1[${PYTHON_USEDEP}]
+	>=dev-python/sentinels-1.0.0[${PYTHON_USEDEP}]
+	>=dev-python/six-1.14.0[${PYTHON_USEDEP}]
+"
+BDEPEND="
+	test? (
+		${RDEPEND}
+		>=dev-python/tox-3.14.4[${PYTHON_USEDEP}]
```

Comment:
yeah need to patch out tox.
it's used by upstreams to run under multiple implementations, eclasses do it for us.